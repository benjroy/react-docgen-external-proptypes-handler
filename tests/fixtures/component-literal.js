import React from 'react';
import { propTypes as propTypesApi, defaultProps as defaultPropsApi } from './api-1';

const ComponentLiteral = props => (<div>component-literal</div>);

ComponentLiteral.propTypes = propTypesApi;
ComponentLiteral.defaultProps = defaultPropsApi;

export default ComponentLiteral;
