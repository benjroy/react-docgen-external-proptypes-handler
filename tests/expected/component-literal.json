{
  "description": "",
  "displayName": "ComponentLiteral",
  "methods": [],
  "props": {
    "api-1-node-required": {
      "type": {
        "name": "node"
      },
      "required": true,
      "description": "Comment for api-1-node-required."
    },
    "api-1-bool": {
      "type": {
        "name": "bool"
      },
      "required": false,
      "description": "Comment for api-1-bool."
    }
  },
  "composes": ["./api-1"]
}
